pkgs/os-specific/linux/open-iscsi/default.nix

```diff
@@ -23,8 +23,17 @@ stdenv.mkDerivation rec {
 
   preConfigure = ''
     sed -i 's|/usr|/|' Makefile
+
+    # Remove blanket -Werror. Fails for minor error on gcc-11.
+    sed -i 's| -Werror | |g' libopeniscsiusr/Makefile usr/Makefile
```

Comment:
We should use substituteInPlace to get a warning if it no longer matches.

Reply:
Oh, good point! I keep forgetting about it. Changed to `substituteInPlace usr/Makefile --replace ' -Werror ' ' '`